lib/fuse.c

```diff
@@ -4870,3 +4885,8 @@ const char *fuse_pkgversion(void)
 {
 	return PACKAGE_VERSION;
 }
+
+struct fuse_config *fuse_get_config(struct fuse *f)
```

Comment:
This seems to be used only by `helper.c`. Why can't `helper.c` include the right header to access `struct fuse`  directly?